sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -697,7 +697,7 @@ public void close() {
             return monoError(logger, new IllegalStateException(
                 String.format(INVALID_OPERATION_DISPOSED_SENDER, "sendMessage")));
         }
-        return getSendLink()
+        return withRetry(getSendLink(), retryOptions, "Can't open link")
```

Comment:
Can we have similar message, like we do in receive link-> `Failed to create a send link.`